backend/lucene/src/main/java/org/hibernate/search/backend/lucene/analysis/impl/LuceneAnalysisComponentFactory.java

```diff
@@ -56,6 +56,10 @@ public Analyzer createAnalyzer(String name, TokenizerFactory tokenizerFactory,
 	public Analyzer createNormalizer(String name,
 			CharFilterFactory[] charFilterFactories, TokenFilterFactory[] filterFactories) {
 		Analyzer normalizer = new TokenizerChain( charFilterFactories, KEYWORD_TOKENIZER_FACTORY, filterFactories );
+		return wrapNormalizer( name, normalizer );
+	}
+
+	public Analyzer wrapNormalizer(String name, Analyzer normalizer) {
 		return new HibernateSearchNormalizerWrapper( normalizer, name );
```

Comment:
Minor nitpicking but it's weird to have the order of parameters reversed between these 2 lines.